CONTRIBUTING.md

```diff
@@ -40,4 +41,31 @@ The repository contains a `justfile` with commands for testing and adding featur
 * **Documentation**: Ensure that your feature is well-documented, including usage instructions and any relevant details.
 * **Testing**: Thoroughly test your feature to ensure it works as expected and does not introduce any issues.
 
+## Using the `pipx-package` Helper
+
+The `pipx-package` helper is a convenient way to install Python packages via pipx. It simplifies the installation process and ensures that the packages are installed in isolated environments.
+
+To use the `pipx-package` helper, follow these steps:
+
+1. **Add the Helper to Your Feature**: In your `devcontainer-feature.json` file, add the `pipx-package` helper to the `installsAfter` array. For example:
+
+    ```json
+    "installsAfter": [
+        "ghcr.io/devcontainers-extra/features/pipx-package",
+        "ghcr.io/devcontainers/features/python"
+    ]
+    ```
+
+2. **Modify the Install Script**: In your `install.sh` script, use the `pipx-package` helper to install the desired Python package. For example:
+
+    ```bash
+    $nanolayer_location \
+        install \
+        devcontainer-feature \
+        "ghcr.io/devcontainers-extra/features/pipx-package:1.1.8" \
```

Comment:
Please, change it to `ghcr.io/devcontainers-extra/features/pipx-package:1.1.9` (that's the latest version)